data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/SelectEntriesProcessorConfig.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.mutateevent;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+public class SelectEntriesProcessorConfig {
+    @NotEmpty
+    @NotNull
+    @JsonProperty("include_keys")
+    private String[] includeKeys;
```

Comment:
Why does this need to be String[]? Why not just make it a List<String>

Reply:
I just copied from DeleteEntries processor code :-) I guess I can make it a list.